.github/workflows/ci-e2e-template.yaml

```diff
@@ -163,4 +163,15 @@ jobs:
 
           echo "script to run: ${script}"
 
-          PW_TEST_CONNECT_WS_ENDPOINT=ws://localhost:5000/ TRUSTIFY_UI_URL=http://localhost:8081 AUTH_REQUIRED=false npm run -w e2e $script
+          PW_TEST_CONNECT_WS_ENDPOINT=ws://localhost:5000/ TRUSTIFY_UI_URL=http://localhost:8081 AUTH_REQUIRED=false npm run -w e2e $script || echo "Tests failed"
+
+      - name: Upload Playwright artifacts
+        if: failure() # only upload if tests failed
+        uses: actions/upload-artifact@v4
+        with:
+          name: playwright-artifacts
+          path: |
+            e2e/test-results/**/*.png
+            e2e/test-results/**/*.zip
```

Comment:
Unless there is a reason, maybe we could upload the whole `e2e/test-results` directory without the need of selecting by file extension